src/components/Input.tsx

```diff
@@ -0,0 +1,32 @@
+import { EyeIcon } from "@heroicons/react/24/solid";
+
+type InputProps = {
+	type: string;
+};
+
+export default function Input(props: InputProps) {
+	let placeholder = "";
+
+	if (props.type === "email") {
+		placeholder = "Email";
+	} else if (props.type === "password") {
+		placeholder = "Password";
+	}
+
+	return (
+		<>
+			<label className="relative">
+				<input
+					className="flex items-center text-center gap-3 pr-8 border border-[#7751C7] focus:border-[#7751C7]focus:border-2 invalid:border-red-500 ring-[#7751C7] rounded-md py-3 px-5 w-full"
+					type={props.type}
+					placeholder={placeholder}
+				/>
+				{props.type === "email" ? (
```

Comment:
no need for a ternary here.
just use the && operator

note: in general if you don't want to render something use null instead of an empty string

Reply:
seems to work